tests/__init__.py

```diff
@@ -0,0 +1,8 @@
+import sys
+
+if sys.version_info[0:2] == (2, 6):
```

Comment:
> Oct 29 2013: Python 2.6.9 final released (security-only)

you don't need to support **python 2.6**

https://github.com/streamlink/streamlink/pull/544
https://www.python.org/dev/peps/pep-0361/#release-schedule

Reply:
True, it was still working until recently - but not now, so I'm not going to fix it. #544 was less about removing support for 2.6, but rather not going out of our way to maintain it.